assets/l10n/app_en.arb

```diff
@@ -27,18 +31,298 @@
+  "@permissionsReadExternalStorageDenied": {
+    "description": "Message for dialog when the user needs to grant permissions for external storage read access."
+  },
+  "actionSheetOptionCopy": "Copy message text",
+  "@actionSheetOptionCopy":  {
+    "description": "Label for copy message text button on action sheet."
+  },
+  "actionSheetOptionShare": "Share",
+  "@actionSheetOptionShare":  {
+    "description": "Label for share button on action sheet."
+  },
+  "actionSheetOptionQuoteAndReply": "Quote and reply",
+  "@actionSheetOptionQuoteAndReply":  {
+    "description": "Label for Quote and reply button on action sheet."
+  },
+  "errorCouldNotFetchMessageSource": "Could not fetch message source",
+  "@errorCouldNotFetchMessageSource": {
+    "description": "Dialog message when the source of a message could not be fetched."
+  },
+  "errorCopyingFailed": "Copying failed",
+  "@errorCopyingFailed": {
+    "description": "Dialog message when copying the text of a message to the users system clipboard failed."
+  },
+  "errorFailedToUploadFile": "Failed to upload file: {filename}",
+  "@errorFailedToUploadFile": {
+    "description": "Label in compose box showing the specified file failed to upload.",
+    "placeholders": {
+      "filename": {
+        "type": "String",
+        "example": "file.txt"
+      }
+    }
+  },
+  "errorFilesTooLarge": "{num, plural, =1{File is} other{{num} files are}} larger than the server's limit of {maxFileUploadSizeMib} MiB and will not be uploaded:\n\n{listMessage}",
+  "@errorFilesTooLarge": {
+    "description": "Error message when attached files are too large in size.",
+    "placeholders": {
+      "num": {
+        "type": "int",
+        "example": "4"
+      },
+      "maxFileUploadSizeMib": {
+        "type": "int",
+        "example": "15"
+      },
+      "listMessage": {
+        "type": "String",
+        "example": "foo.txt\nbar.txt"
+      }
+    }
+  },
+  "errorFilesTooLargeTitle": "{num, plural, =1{File} other{Files}} too large",
+  "@errorFilesTooLargeTitle": {
+    "description": "Error title when attached files are too large in size.",
+    "placeholders": {
+      "num": {
+        "type": "int",
+        "example": "4"
+      }
+    }
+  },
+  "errorLoginInvalidInputTitle": "Invalid input",
+  "@errorLoginInvalidInputTitle": {
+    "description": "Error dialog title for login dialog when input is invalid."
+  },
+  "errorLoginFailed": "Login failed",
+  "@errorLoginFailed": {
+    "description": "Error dialog title when login for a Zulip server fails."
+  },
+  "errorMessageNotSent": "Message not sent",
+  "@errorMessageNotSent": {
+    "description": "Message for compose box when a message could not be sent."
+  },
+  "errorMessageDoesNotSeemToExist": "That message does not seem to exist.",
+  "@errorMessageDoesNotSeemToExist": {
+    "description": "Error message when loading a message that does not exist."
```

Comment:
nit: This description gets introduced as one thing by an earlier commit, then edited by an otherwise-unrelated later commit.